tests/postgres_tests/models.py

```diff
@@ -214,3 +214,19 @@ class HotelReservation(PostgreSQLModel):
     end = models.DateTimeField()
     cancelled = models.BooleanField(default=False)
     requirements = models.JSONField(blank=True, null=True)
+
+
+if connection.vendor == "postgresql":
+
+    class NullableIntegerRangeCheckConstraintModel(models.Model):
```

Comment:
Inherit from `PostgreSQLModel` instead of `using connection.vendor`.

Reply:
This model is only used in a single test. I'd re-use an existing model e.g. `AggregateTestModel`, and add a test method to the `postgres_tests/test_constraints.py`. Maybe:
```python
    def test_validate_range_lower_upper(self):
        constraint = CheckConstraint(
            check=Q(ints__startswith__gte=0) & Q(ints__endswith__lte=99),
            name="ints_range_lower_upper",
        )
        msg = f"Constraint “{constraint.name}” is violated."
        with self.assertRaisesMessage(ValidationError, msg):
            constraint.validate(RangesModel, RangesModel(ints=(-1, 20)))
        with self.assertRaisesMessage(ValidationError, msg):
            constraint.validate(RangesModel, RangesModel(ints=(0, 100)))
        constraint.validate(RangesModel, RangesModel(ints=(0, 99)))
```

Reply:
Assuming you mean the `RangesModel` 😊?… yep if it doesn't detract in any way from other tests 👍